sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -1,7 +1,16 @@
 ## Release History
 
-## 4.12.0-beta.1 (Unreleased)
+### 4.12.0 (2021-02-09)
+#### New Features
```

Comment:
Generic comments: How about include PR reference in the release notes?

Reply:
We don't include PR references in release notes, its against azure java SDK guidelines.